deployment/vagrant/bootstrap.sh

```diff
@@ -9,62 +9,18 @@ if [ ! -e "~/.firstboot" ]; then
   echo "grub-pc grub-pc/install_devices multiselect     /dev/sda" | sudo debconf-set-selections
 
   # Install prerequisite tools
-  echo "Installing prerequisites"
+  echo "Installing docker"
+  curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
   sudo apt-get update
-  sudo apt-get install -y git
-  git config --global core.autocrlf input
-
-  # Setting up ssh & passwordless sudo
-  ssh-keygen -f /home/vagrant/.ssh/id_rsa -N '' -t rsa
-  cat /home/vagrant/.ssh/id_rsa.pub >> /home/vagrant/.ssh/authorized_keys
-  chmod og-wx /home/vagrant/.ssh/authorized_keys
-  echo "NoHostAuthenticationForLocalhost yes" | tee -a /home/vagrant/.ssh/config
-  chmod 600 ~/.ssh/config
-  echo "vagrant ALL=(ALL:ALL) NOPASSWD: ALL" | sudo tee -a /etc/sudoers
+  sudo apt-get install -yqq docker-ce
```

Comment:
Do you not need to do that `sudo useradd ...` thing?

Reply:
I didn't have to do any useradding